docs/Fluent-Schema.md

```diff
@@ -0,0 +1,118 @@
+<h1 align="center">Fastify</h1>
+
+## Fluent Schema
+
+The [Validation and Serialization](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md)
+has explained all the parameter accepted by Fastify to set a JSON Schema Validation, to validates
+the input, and a JSON Schema Serialization to optimize the output.
+
+To set up the schemas of our Fastify application, we can use [`fluent-schema`][fluent-schema-repo]
+to simplify this task.
+
+### Basic settings
+
+```js
+const S = require('fluent-schema')
+
+const bodyJsonSchema = S.object()
+  .prop('someKey', S.string())
+  .prop('someOtherKey', S.number())
+  .prop('requiredKey', S.array().maxItems(3).items(S.integer()).required())
+  .prop('nullableKey', S.mixed([S.TYPES.NUMBER, S.TYPES.NULL]))
+  .prop('multipleTypesKey', S.mixed([S.TYPES.BOOLEAN, S.TYPES.NUMBER]))
+  .prop('multipleRestrictedTypesKey', S.oneOf([S.string().maxLength(5), S.number().minimum(10)]))
+  .prop('enumKey', S.enum(['John', 'Foo']))
```

Comment:
`enum` is one of the main reason to use fluent-schema so I would show the fact you can use an array previously declared 

```
const MY_KEYS = {
 KEY1: 'ONE'
 KEY2: 'TWO'
}
...
.prop('enumKey', S.enum(Object.values(MY_KEYS)))
```

Reply:
Done, thank you for the suggestions 👍